lib/seattleflu/id3c/api/routes.py

```diff
@@ -35,7 +38,16 @@ def get_metadata_v1():
 
 @api_v2.route("/shipping/augur-build-metadata", methods = ['GET'])
 @authenticated_datastore_session_required
-def get_metadata_v2(session):
+def get_metadata_v2():
+    """
+    Mark old endpoint as 410 Gone
+    """
+    return "use /v3/shipping/augur-build-metadata instead", 410
```

Comment:
I believe we don't actually want to mark this endpoint as gone, but rather delete the v2 endpoint after we merge the related backoffice and website PR's (linked in top-level comment). 

Seeking feedback here.

Reply:
I think 410 Gone is fine and slightly more useful if someone runs an old saved command against those endpoints. But removing them after we think all uses are updated would also be ok, and in that case we'd want to remove both the v1 and v2 endpoints.

Reply:
Ah, sorry @tsibley, I was unclear. 
I am ok with marking the v2 endpoint as "gone" but only after we merge the dependent PRs linked at the top. 
I should not have used the word "delete"

Reply:
Essentially, I am considering reverting this part of the commit to keep the v2 endpoint functioning for now.

Reply:
Ah, I see. Yep. Although those PRs at the top will require the `augur-build-exporter` role updates in this PR.

Reply:
Right, so we should merge this PR first. 
I don't believe the `augur-build-exporter` changes only add privileges, not revoke any.

Reply:
@tsibley In my latest fixed-up commit, the v2 endpoint is back to its original and is no longer marked as gone.

Reply:
Agh I just realized that doesn't work because the datastore function is updated to use v3. Unsure now how to proceed.

Reply:
Thomas and I discussed in person the idea to create a new, generic function that exports all rows from any qualified (schema, table) couple to ndjson. This exists as a new commit in this PR.